synfig-studio/src/gui/canvasview.cpp

```diff
@@ -298,6 +302,26 @@ class studio::CanvasViewUIInterface : public UIInterface
 	virtual bool
 	amount_complete(int current, int total)
 	{
+		float temp = (float)current/(float)total;
+		
+		if(temp >= 1.0)
+		{
+			view->statusbar->show();
+			view->progressbar->hide();
+		}
+
+		if(temp != cur_progress)
```

Comment:
This is not quite the correct option for comparing floats. 
Better to use something like:

```
if (fabs(temp-cur_progress) >= 0.1) {
}
```

Reply:
Yes, and I suggest to use smaller compare value:
```
if (fabs(temp-cur_progress) >= 0.01) {
}
```

Reply:
okay, Done changes

Reply:
Also, should it be `else if` instead of `if`?

If progress value changes from 1.1 to 1.2, then this condition will be triggered and progressbar will appear. ^__^